### ISSUE TITLE
Too many requests, error code 429 on Magento 2 environments

### ISSUE THREAD

**Issue by cream-julian** [ByteInternet/hypernode-vagrant#193]
Using the default configuration for Magento 2 projects, we're getting errors in dev tools console that for most requests have response code 429.

I have followed the knowledgebase to resolve this issue. We did not succeed in resolving this.
https://support.hypernode.com/knowledgebase/resolving-429-many-requests/

This shouldn't happen on development environments.

Vagrant version: 1.9.5
Hypernode_Xenial version: 4917

**Comment by andreas-bytenl** [comment]
I agree with this. We might look into finding a way to re-structure our configuration to disable this feature for Vagrant specifically.

In the KB, did you follow the step involving `/data/web/nginx/http.conn_ratelimit`? If you did and it didn't make a difference, we may have to double-check that the KB isn't missing anything.
